javascript/javascript1/week2/Flight booking fullname function.js

```diff
@@ -0,0 +1,14 @@
+const fullname1 = 'Anna',
+      fullname2 = 'Datskevych';
+
+
+function getFullname(firstname, surname, useFormalName=false, male=true) {
+    if (useFormalName & male) {
+        return `Lord ${firstname} ${surname}`;
+    }else if (useFormalName & male == false) {
+        return `Ms ${firstname} ${surname}`;
+    }
+    return `${firstname} ${surname}`;
+}
+
+console.log(getFullname(fullname1, fullname2, true, false));
```

Comment:
This works smoothly! but what if I forget to type in my name? 
can you maybe add a check and a prompt to enter a full name?